client/components/formatted-header/README.md

```diff
@@ -3,6 +3,8 @@ FormattedHeader (JSX)
 
 This component displays a header and optional sub-header.
 
+When the `responsive` flag is set, the header renders in a compact way on smaller screen sizes.
```

Comment:
I have second thoughts about the name of this flag. Considering that it makes the header "compact" does it make sense to rename it to `compact`? "Responsive" seems to be too broad in my head.